src/View/mainpage/StructureTab.py

```diff
@@ -1,15 +1,16 @@
 import csv
+import pydicom
 from pathlib import Path
 from random import randint, seed
-
 from PySide6 import QtWidgets, QtGui, QtCore
 from PySide6.QtCore import Qt
 
 from src.Controller.ROIOptionsController import ROIDelOption, ROIDrawOption
 from src.Model import ImageLoading
 from src.Model.GetPatientInfo import DicomTree
 from src.Model.PatientDictContainer import PatientDictContainer
-from src.Model.ROI import ordered_list_rois, get_roi_contour_pixel, calc_roi_polygon, transform_rois_contours
+from src.Model.ROI import ordered_list_rois, get_roi_contour_pixel, calc_roi_polygon, transform_rois_contours, \
```

Comment:
PEP 8 compliance - line too long (also lines 357, 361, 369-373, 377, 381-382, 386, 393, 401, 403, and 407-409).